elixir/apps/web/lib/web/live/settings/dns.ex

```diff
@@ -1,7 +1,113 @@
 defmodule Web.Settings.DNS do
   use Web, :live_view
 
+  alias Domain.Config
+
+  defp remove_errors(changeset, field, message) do
+    errors =
+      changeset.errors
+      |> Enum.filter(fn err ->
+        case err do
+          {^field, {msg, _}} ->
+            if msg == message, do: false, else: true
+
+          {_, _} ->
+            true
+        end
+      end)
```

Comment:
```suggestion
      Enum.filter(changeset.errors, fn 
        {^field, {^message, _}} -> true
        {_field, _metadata} -> false
      end)
```

Reply:
Thank you for this suggestion, I really like how this makes the code both more compact and easier to understand. 🙏

Reply:
I just realized though, don't you still need the `case` statement?